beacon-chain/state/interfaces.go

```diff
@@ -179,6 +180,12 @@ type ReadOnlyEth1Data interface {
 	Eth1DepositIndex() uint64
 }
 
+// ReadOnlyExits defines a struct which only has read access to Exit related methods.
+type ReadOnlyExits interface {
+	ExitBalanceToConsume() (math.Gwei, error)
+	EarliestExitEpoch() (primitives.Epoch, error)
+}
```

Comment:
This stuff is never used.

Reply:
used in tests

Reply:
in tests you can just cast to the appropriate type and read the field directly